docs/developer_guide/self-hosted.md

```diff
@@ -102,10 +102,10 @@ At any point, if any of the services fails to launch or something seems broken,
 1. You can run `docker-compose ps`  to see all running services. You should see that services `fluree`, `athens`, and `nginx` are "up" and/or "healthy". ![docker-ps-output](https://user-images.githubusercontent.com/8952138/142656719-21c54b94-8f50-4091-9044-a72bac1988a2.png)
 2. Additionally, you can use `docker-compose logs SERVICE_NAME` to inspect what the problem is further.
 
-Pick a [release](https://github.com/athensresearch/athens/releases) you'd like to use and download the `docker-compose.yml`. For example, for `v2.0.0-beta.3`:
+Pick a [release](https://github.com/athensresearch/athens/releases) you'd like to use and download the `docker-compose.yml`. For example, for `v2.0.0-beta.6`:
 
 ```sh
-curl -L -o docker-compose.yml https://github.com/athensresearch/athens/releases/download/v2.0.0-beta.3/docker-compose.yml
+curl -L -o docker-compose.yml https://github.com/athensresearch/athens/releases/download/v2.0.0-beta.6/docker-compose.yml
```

Comment:
`beta.3` didn't actually work for me but `beta.6` did so I thought this change would help others